packages/core/src/App/Containers/Modals/crypto-transaction-processing-modal/crypto-transaction-processing-modal.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useDevice } from '@deriv-com/ui';
+import { Button, MobileDialog, Modal } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { CryptoTransactionProcessingModalContent } from './crypto-transaction-processing-modal-content';
+import './crypto-transaction-processing-modal.scss';
+
+const CryptoTransactionProcessingModal = observer(() => {
+    const { isDesktop } = useDevice();
+    const { ui } = useStore();
+    const { should_show_crypto_transaction_processing_modal, setShouldShowCryptoTransactionProcessingModal } = ui;
+
+    const onCloseModal = () => {
+        setShouldShowCryptoTransactionProcessingModal(false);
+    };
+
+    return (
+        <React.Fragment>
+            {isDesktop ? (
+                <Modal
+                    is_open={should_show_crypto_transaction_processing_modal}
+                    className='crypto-transaction-processing-modal'
+                    title=' '
+                    toggleModal={onCloseModal}
+                    height='440px'
+                    width='440px'
+                    has_close_icon
+                    should_header_stick_body={false}
+                >
+                    <CryptoTransactionProcessingModalContent />
+                    <Modal.Footer has_separator>
+                        <Button large primary onClick={onCloseModal}>
+                            OK
```

Comment:
Shouldn't we translate this text?

Reply:
@niloofar-deriv sure, you are right, forgot about this